src/main.js

```diff
@@ -18,16 +17,9 @@ var saveInputTitle = document.querySelector(".user-title");
 var saveInputDesc1 = document.querySelector(".user-desc1");
 var saveInputDesc2 = document.querySelector(".user-desc2");
 var savedCoverSect = document.querySelector(".saved-covers-section");
-var currentCover = new Cover();
 
-var savedCovers = [
-  new Cover(
-    "http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg",
-    "Sunsets and Sorrows",
-    "sunsets",
-    "sorrows"
-  ),
-];
+var currentCover = new Cover();
```

Comment:
Please add lines 23 and 24 to top of page as they are global variables.

Reply:
Done!